org.eclipse.jdt.debug.tests/tests/org/eclipse/jdt/debug/tests/breakpoints/TestToggleBreakpointsTarget8.java

```diff
@@ -124,4 +120,26 @@ public void testInterfaceLineBreakpoint() throws Exception {
 		}
 	}
 
+	/**
+	 * Tests that a line breakpoints in an lambda expression works.
+	 */
+	public void testLineBreakpointInsideLambda() throws Exception {
+		Listener listener = new Listener();
+		IBreakpointManager manager = getBreakpointManager();
+		manager.addBreakpointListener(listener);
+		try {
+			Path path = new Path("java8/ClassWithLambdas.java");
+			final int lineNr = 35; // 0 based offset in document line numbers
```

Comment:
it is not so easy to understand where that breakpoint hits. idea: how about adding a comment in the source example "// breakpoint", and search that line here?

Reply:
In general a good idea but all other existing tests here use the implemented approach so I suggest to keep it "as is". I also think that the comment "0 based offset in document line numbers" should be clear enough for developers to understand where that breakpoint hits.